crates/tinyvec-embed/src/db.rs

```diff
@@ -244,6 +252,18 @@ impl Value {
     }
 }
 
+impl TryInto<usize> for &Value {
+    type Error = Error;
+
+    fn try_into(self) -> Result<usize> {
+        if let Value::Number(n) = self {
+            Ok(n.clone() as usize)
+        } else {
+            Err(Error::ValueNotUsize(self.to_owned()))
```

Comment:
```suggestion
            Err(Error::ValueNotUsize(self))
```
`fn try_into(self)` takes an owned value of `self`, I don't think you need to call `to_owned` here

Reply:
Same comment as above, working on &Value requires the to_owned in my tests